src/Analyzers/CSharp/Tests/UseCollectionInitializer/UseCollectionInitializerTests.cs

```diff
@@ -154,6 +154,122 @@ void M(int[] x)
                 """);
         }
 
+        [Fact]
+        public async Task TestOnVariableDeclarator_If1()
+        {
+            await TestInRegularAndScriptAsync(
+                """
+                using System.Collections.Generic;
+
+                class C
+                {
+                    void M(bool b)
+                    {
+                        var c = [|new|] List<int>();
+                        [|c.Add(|]1);
+                        if (b)
+                            c.Add(2);
+                    }
+                }
+                """,
+                """
+                using System.Collections.Generic;
+
+                class C
+                {
+                    void M(bool b)
+                    {
+                        var c = new List<int>
+                        {
+                            1
+                        };
+                        if (b)
+                            c.Add(2);
```

Comment:
these tests validate that for the *collection INITIALIZER* space we don't do anything special.  The new form is for the *collection EXPRESSION* case.